catalog/transformer/catalog-transformer-pdf/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -12,9 +12,36 @@
  *
  **/
  -->
-<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+           xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0">
 
-    <bean id="pdfTransformer" class="ddf.catalog.transformer.input.pdf.PdfInputTransformer"/>
+    <bean id="pdfMetacardType" class="ddf.catalog.data.impl.MetacardTypeImpl">
+        <argument value="pdf"/>
+        <argument>
+            <list>
+                <bean class="ddf.catalog.data.impl.types.AssociationsAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.ContactAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.MediaAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.ValidationAttributes"/>
```

Comment:
@glenhein - looks like the Topic category is missing here and possible DateTime.